internal/config/config_test.go

```diff
@@ -1351,3 +1355,59 @@ func Test_mustBindEnv(t *testing.T) {
 		})
 	}
 }
+
+type mockURLOpener struct {
+	bucket *blob.Bucket
+}
+
+// OpenBucketURL opens a blob.Bucket based on u.
+func (c *mockURLOpener) OpenBucketURL(ctx context.Context, u *url.URL) (*blob.Bucket, error) {
+	for param := range u.Query() {
+		return nil, fmt.Errorf("open bucket %v: invalid query parameter %q", u, param)
+	}
+	return c.bucket, nil
+}
+
+func TestRemoveConfig(t *testing.T) {
```

Comment:
```suggestion
func TestGetConfigFile(t *testing.T) {
```

would ☝🏻 be more accurate of a test name?

Reply:
or `TestRemoteConfig`